app/controllers/api/databases.php

```diff
@@ -542,46 +547,42 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_COLLECTION_LIST)
     ->param('databaseId', '', new UID(), 'Database ID.')
+    ->param('queries', [], new Collections(), 'Array of query strings generated using the Query class provided by the SDK. [Learn more about queries](https://appwrite.io/docs/databases#querying-documents). Maximum of ' . APP_LIMIT_ARRAY_PARAMS_SIZE . ' queries are allowed, each ' . APP_LIMIT_ARRAY_ELEMENT_SIZE . ' characters long. You may filter on the following attributes: ' . implode(', ', Collections::ALLOWED_ATTRIBUTES), true)
```

Comment:
we might wanna alias import this, `new Collections()` doesn't tell a story.

Reply:
I wanted to suffix them all with `Query` or `Validator`. The problem then is that we are duplicating it since the file path already explains what it is related to. I noticed different team members have different opinions on this. Making myself note to address this in future (since it's not critical for the release)